__tests__/server/question-routes.spec.js

```diff
@@ -0,0 +1,21 @@
+const supertest = require('supertest');
+const app = require('../../server/server');
+const request = supertest(app);
+
+describe('GET /api/questions', () => {
+  it('should print questions in json format when request is successful', async () => {
+    const res = await request.get('/api/questions');
+    // The current bruteforce way of checking it is to see if there are more than 8 questions returned.
+    // TODO: should change the following after we implement the create question function.
+    expect(res.status).toBe(200);
+    expect(res.body.length).toBeGreaterThan(8);
+  });
+});
+

```

Comment:
Did some thinking and realized 2 more tests you should add - sorry didn't mention this earlier (might mean renaming this test above)

1. Check that the object returned has fields 'ID' and 'text'
2. Check that the text of the first object/question in the returned array is "Who/what influenced or inspired you to do what you do?" which won't be changed